apps/enc.c

```diff
@@ -360,8 +360,8 @@ int enc_main(int argc, char **argv)
                 char prompt[200];
 
                 BIO_snprintf(prompt, sizeof(prompt), "enter %s %s password:",
-                        OBJ_nid2ln(EVP_CIPHER_nid(cipher)),
-                        (enc) ? "encryption" : "decryption");
+                             EVP_CIPHER_description(cipher),
```

Comment:
This can be (and currently will be) NULL, won't it?

Reply:
(and maybe I should leave this to you, then... :wink:)

Reply:
I agree with using EVP_CIPHER_name.

Reply:
So I've added the change to EVP_CIPHER_name into #14703

Reply:
Okie, dropped